docs/standards/nft-2.0/02-LSP7-Digital-Asset.md

```diff
@@ -1,5 +1,6 @@
 ---
 sidebar_label: 'LSP7 - Digital Asset'
+sidebar_position: 3
```

Comment:
filename is `02-LSP7...` but sidebar position is 3 - weird?

[02-LSP7-Digital-Asset.md](https://github.com/lukso-network/docs/pull/111/files#diff-d8513e61780c113e2cbe1b4fc8d8bbd0dcd5477eda17e377c6f87bcfc983afae)